internal/app/services/loginService/loginService.go

```diff
@@ -16,11 +17,11 @@ type Response struct {
 }
 
 func Execute(ctx context.Context, req Request) (*Response, error) {
-	err := users.VerifyPassword(ctx, req.Email, req.Password)
+	err := userModel.VerifyPassword(ctx, req.Email, req.Password)
```

Comment:
We are currently doing two calls to `GetUserByEmail`. I don't think its necessary.
We can `GetUserByEmail` first, if no error, if we change the `VerifyPassword` to take in a user and a password then all we need for the verify function is `return bcrypt.CompareHashAndPassword([]byte(user.Password), []byte(password))`.

Reply:
Oh and then afterwards, would need to move `VerifyPassword` to `components/users` if we go through with the change.

Reply:
Yeah that makes more sense